package.json

```diff
@@ -33,7 +33,7 @@
     "express": "^4.16.4",
     "express-validator": "^6.6.1",
     "front-matter": "^4.0.2",
-    "glob": "^7.1.4",
+    "glob": "^7.2.0",
```

Comment:
Why is the bump to gulp needed?

Reply:
This is a bump to `glob` - not necessary for this work, it just bumps it to the latest version

Reply:
Oh yeah, force of habit 😅 

It might be good to split out the bump into a separate commit or PR, just in case it happens to break something?

Reply:
Good shout, I've removed it from this PR and I'll raise a separate PR for that